sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Release History
 
+## 1.8.1 (2023-04-06)
+
+### Bugs Fixed
+
+- [[#4213]](https://github.com/Azure/azure-sdk-for-cpp/issues/4213) Fixed a bug where `Host` request header is not set for non-default port (80, 443).
+- [[#4443]](https://github.com/Azure/azure-sdk-for-cpp/issues/4443) Fixed potentially high CPU usage on Windows.
+
+### Other Changes
+
+- Libcurl transport doesn't add `Content-Length` request header for GET/HEAD/DELETE requests anymore.
+
```

Comment:
@RickWinter, it includes `Other Changes/Content-Length header` because it is the part of the same commit as `Bugs Fixed/Host header` (https://github.com/Azure/azure-sdk-for-cpp/pull/4515/commits/084a9f2d0b4972e952560fdf6b00e2a9993e308f#diff-cc1b46c667702f97b0f4104c7637ccca9da686d2df0cd9e47a39f12ced659897)